src/gobupload/compare/enrich.py

```diff
@@ -194,6 +194,9 @@ def _autoid(storage, data, specs, column, assigned):
 
     current = _get_current_value(storage, data, specs, column, assigned)
     if current:
+        # Insert current value and adjust last assigned value if required
+        data[column] = current
+        _update_last_assigned(data, specs, column, assigned)
```

Comment:
What happens when the auto id range differs from the id's already present in de database? For example, when a list of objects of range AAAxxxxxx is imported, and the autoid is of range BBBxxxxxx. It seems like the AAA range is now included in last_assigned. I don't have the complete picture at this moment, but I have the feeling this will cause problems. (The BAG autoid's are of a different range than the range assigned by DIVA, that's why)

Reply:
update_last_assigned checks if the pattern is matched. So it should not conflict with ids that do not match the template pattern

Reply:
Ok cool, then it shouldn't be a problem :)